### PR TITLE
Add v3 release blog post

### PR DESCRIPTION
**Pull request by mauromorales**

### CODE REVIEW

File: content/en/blog/release-v3.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+* 🐛 reset from the GRUB menu on Alpine amd64, gets stuck in an endless loop [#2136](https://github.com/kairos-io/kairos/issues/2136)
+
+## Deprecation Warnings
+
+Reading of `/etc/elemental/config.yaml` was broken for a bit but should be fixed in v3, but it will be eventually deprecated in one of the upcoming releases. If you're making use of it, please move this configuration to `/etc/kairos/config.yaml`
+
+
+## Flavor Updates
+
+* UKI Ubuntu and non-UKI Ubuntu differ a bit. This is because we need to make UKI Ubuntu as slim as possible. We will probably continue working towards this goal
```
Comment (jimmykarily) [inline content/en/blog/release-v3.md:44]: Maybe mention the size limitations in UKI booting to justify why we needed to slim it down. Also a `.` is missing after `goal`.

**Review by jimmykarily — APPROVED** [review]
Good job @mauromorales !

File: content/en/blog/release-v3.md (hunk 1 above)
Comment (mauromorales) [reply content/en/blog/release-v3.md:44]: Whoops saw this after merging will fix in a different commit

Comment (mauromorales) [reply content/en/blog/release-v3.md:44]: done

**Pull request merged**